### ISSUE TITLE
typescript-graphql-request plugin doesn't include the errors key

### ISSUE THREAD

**Issue by david-wb** [dotansimha/graphql-code-generator#7738]
After upgrading from version 4.4.3 to version 4.4.5 of the plugin, the `errrors` key is no longer available in the response object, even when using the `rawRequest: true` option.

<img width="280" alt="Screen Shot 2022-04-05 at 10 47 18 AM" src="https://user-images.githubusercontent.com/1855225/161805646-3fca2a39-7b38-4a29-bc31-4e3260024c20.png">

**Comment by chrisdhanaraj** [comment]
Looks like @P4sca1 intentionally removed that in https://github.com/dotansimha/graphql-code-generator/pull/7676

**Comment by david-wb** [comment]
Is that a bug? How can I get the `.errors` now then? Definitely need that.

**Comment by chrisdhanaraj** [comment]
It's a bug in `graphql-request` (it never really worked - you always had to try/catch to get it to function) - here's an Open PR that fixes that in graphql-request.

https://github.com/prisma-labs/graphql-request/pull/230

**Comment by P4sca1** [comment]
Yes, it has been removed intentionally. The typings used to be wrong. Code that uses the errors property for error handling is not working as intended, because the property is always undefined and an error is thrown instead.

**Comment by awinograd** [comment]
graphql-request 5.0 has an [errorPolicy](https://github.com/prisma-labs/graphql-request#errorpolicy) option which returns errors when using rawRequest. Perhaps the errors field can be added back as a plugin option?
